components/states/DiverInformationState.jsx

```diff
@@ -1,62 +1,81 @@
 import React from "react";
 import { useActor } from "@xstate/react";
-import { Button } from "@mui/material";
-import { useForm } from "react-hook-form";
-import isEmail from "validator/lib/isEmail";
-import isNumeric from "validator/lib/isNumeric";
+
+import { Button, TextField } from "@mui/material";
 
 import { STATE_ACTIONS } from "../../utils/state-machine";
 import StatePage from "./shared/StatePage";
 import StateTitle from "./shared/StateTitle";
 import { MyContext } from "../ReservationController";
-import FormTextField from "../shared/FormTextField";
+import { statesText } from "../../utils/app-text";
 
 const DiverInformationState = () => {
-  const machine = React.useContext(MyContext);
+  const context = React.useContext(MyContext);
+
+  const [name, setName] = React.useState("");
+  const [lastName, setLastName] = React.useState("");
+  const [email, setEmail] = React.useState("");
+  const [age, setAge] = React.useState();
 
-  const [, send] = useActor(machine);
+  const [, send] = useActor(context.service);
 
-  const { handleSubmit, control } = useForm();
-  const handleSubmitForm = ({ name, lastName, email, age }) => {
-    send({
-      type: STATE_ACTIONS.CREATE_SUPABASE_RESERVATION,
-      value: { name, lastName, email, age },
-    });
-  };
   return (
     <StatePage>
-      <StateTitle title="Diver Information" />
-      <form onSubmit={handleSubmit(handleSubmitForm)}>
```

Comment:
you are removing this... we should leave the `form`, that is what allows us to submit the info right?